<!DOCTYPE html>
<html>
<head>

  <meta charset="UTF-8">
  <title>Research Projects</title>
  <meta name="viewport" content="width=device-width">

  <!--[if lt IE 9]>
    <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
  <![endif]-->

  <link href="/assets/css/style.css" rel="stylesheet" />
  <link href="/assets/css/colors-light.css" rel="stylesheet" />



</head>

<body>



  <header id="header">
    <h1><a href="/"></i> Xiang Ji's Webpage</a></h1>
    <p>A place for some footprints</p>
  </header>



  <div id="page">



    <div id="sidebar">
      <nav>
        <ul>
          <li><a href="/">Home</a></li>
<!--          <li><a href="/archives">Archive</a></li>
-->
          <li><a href="/Courses">Courses Taken</a></li> 
          <li><a href="/Publication">Projects</a></li>
          <li><a href="/CV">CV</a></li>
<!--
-->

        </ul>
      </nav>
    </div>



    <div id="content">
      <h2 id="phd-research">PhD Research</h2>

<p><strong><em>Disentangling IGC tract length and initiation rate</em></strong> project, NC State University, 2017</p>

<p>Advisor: Dr. Jeff Thorne, Bioinformatics Research Center, NC State University</p>

<p>A shortcoming of our original approach (see the <em>Modeling Gene Conversion</em> project, also as in <a href="http://mbe.oxfordjournals.org/content/early/2016/06/11/molbev.msw114.short?rss=1">Ji et al. 2016</a>) towards quantifying interlocus gene conversion (IGC) was its inability to infer the tract length distribution of (fixed) IGC events.  I developed a method to address this and am beginning to apply it. This ongoing project can be found <a href="https://github.com/xji3/YeastIGCTract">here on github</a>.</p>

<p><strong><em>IGC and gene duplication loss histories of primate ADH1 genes</em></strong> project, NC State University, 2016</p>

<p>Advisor: Dr. Jeff Thorne, Bioinformatics Research Center, NC State University</p>

<p>Motivated by primate ADH1 data, I extended our method (as in <a href="http://mbe.oxfordjournals.org/content/early/2016/06/11/molbev.msw114.short?rss=1">Ji et al. 2016</a>) so that up to 6 paralogs per genome can be considered. I am improving my software to answer interesting biological questions (e.g., permitting asymmetry among paralogs in tendency to serve as IGC donor, investigating how distance between tandem paralogs affects IGC, summarizing how paralog divergence changes IGC). I am also considering different gene duplication loss histories and how incorporating IGC could help with orthologous mapping. This ongoing project can be found <a href="https://github.com/xji3/ADH1Genes">here on github</a>.</p>

<p><strong><em>Modeling Gene Conversion</em></strong> project, NC State University, 2014</p>

<p>Advisor: Dr. Jeff Thorne, Bioinformatics Research Center, NC State University</p>

<p>Interlocus gene conversion (IGC) homogenizes repeats.  While genomes can be repeat-rich, the evolutionary importance of IGC is poorly understood, largely due to a lack of statistical tools.  We developed a strategy for adding IGC to widely-used probabilistic models for sequence change.  In 14 groups of yeast ribosomal protein genes, we estimated the percentage of codon substitutions that originate with IGC rather than point mutation.  We found values ranging from 20% to 38%.  Our results are consistent with an important role for IGC in the evolution of each of the 14 gene families.  Our <a href="https://academic.oup.com/mbe/article/33/9/2469/2579387/A-Phylogenetic-Approach-Finds-Abundant-Interlocus">paper</a> is published in MBE (<em>Molecular Biology and Evolution</em>).</p>

<p>Here is the <a href="https://github.com/xji3/JGT_MBE_2016">github repository</a> of the publication. This <a href="https://github.com/xji3/IGCexpansion">github repository</a> keeps my under development software code.</p>

<p><strong><em>Solvent Accessibility</em></strong> project, NC State University, 2013</p>

<p>Advisor: Dr. Jeff Thorne, Bioinformatics Research Center, NC State University</p>

<p>This was a ‘left-over’ project that I helped finished. I extracted solvent accessibility information from protein coding genes in several species (human, mouse and Ecoli). Another senior PhD student built probabilistic model based on these data. Details of this project can be found in this <a href="https://github.com/xiang-ji-ncsu/xiang-ji-ncsu.github.io/raw/master/Publication/Roles%20of%20Solvent%20Accessibility%20and%20Gene%20Expression%20in%20Modeling%20Protein%20Sequence%20Evolution.pdf">paper</a> which I co-authored.</p>

<p>=======================================================</p>

<h2 id="master-thesis">Master Thesis</h2>

<p><strong><em>Laser Interference Lithography</em></strong> project, NC State University, 2012</p>

<p>Advisor: Dr. John Muth, EE Department, NC State University</p>

<p>The goal was to develop a fabrication process using the mask-free laser interference lithography to easily introduce periodic pattern onto devices. We wanted to use it for gas sensor fabrication to increase the surface to volumn ratio which primarily determines gas-sensor’s sensitivity. Details can be found in my <a href="https://github.com/xiang-ji-ncsu/xiang-ji-ncsu.github.io/raw/27914ae129b83c237d03d68fec002646d1163f69/Publication/Laser%20Interference%20Lithography%20for%20Fabrication%20of%20Gas%20Sensors.pdf">ugly-written thesis</a>.</p>

<p>Here are some pictures for a taste of what I did.</p>

<p><img src="https://github.com/xiang-ji-ncsu/xiang-ji-ncsu.github.io/raw/master/images/Work%20in%20cleanroom.png" /></p>

<table>
  <thead>
    <tr>
      <th style="text-align: center">Periodic lines on sample</th>
      <th style="text-align: center">Gas sensor prototype</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center"><img src="https://github.com/xiang-ji-ncsu/xiang-ji-ncsu.github.io/raw/master/images/Periodic%20Lines.png" alt="Periodic lines on sample" /></td>
      <td style="text-align: center"><img src="https://github.com/xiang-ji-ncsu/xiang-ji-ncsu.github.io/raw/master/images/Gas%20sensor%20prototype.png" alt="Gas sensor prototype" /></td>
    </tr>
  </tbody>
</table>

<p>=======================================================</p>

<h2 id="undergraduate-research">Undergraduate Research</h2>

<p><strong><em>SPP source</em></strong> project, Peking University, 2011</p>

<p>Advisor: Dr. Zhiping Zhou, EECS Department, Peking University</p>

<p>I did my undergraduate thesis in this lab. Part of the work I did was to find a good doping location for SPP source design. The idea was to imbed light source inside a plasmon waveguide. You could find details in this published <a href="https://github.com/xiang-ji-ncsu/xiang-ji-ncsu.github.io/raw/27914ae129b83c237d03d68fec002646d1163f69/Publication/Effect%20of%20dipole%20location%20on%20profile%20properties%20of%20symmetric%20surface%20plasmon%20polariton%20mode%20in%20Au-Al2O3-Au%20waveguide.pdf">paper</a> which I co-authored.</p>

<p><strong><em>Plasmonic Lens</em></strong> project, Peking University, 2010</p>

<p>Advisor: <a href="http://www.phy.pku.edu.cn/~zhangjs/index.html">Dr. Jiasen Zhang</a>, Physics Department, Peking University</p>

<p>The Goal was to design, fabricate and test rectangular shape plasmonic lense composit of nano-scaled slots on a 200nm thin gold film on glass substrate being able to generate sharper focus. This work stays unpublished.</p>

<p>Due to detection limit, we could not provide direct proof that our device worked by the fact that our image detection system was only sensitive to XY plane (parallel to the device) of the optical field whereas the focus intensity mainly lied in Z direction. If possible, a near field optic probe may detect the focus point and give a direct proof. I show my simulated light intensity distribution at XY plane of the focus point together with experimental result below to give you an idea.</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center">XY intensity plane simulated</th>
      <th style="text-align: center">intensity experimentally detected</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center"><img src="https://github.com/xiang-ji-ncsu/xiang-ji-ncsu.github.io/raw/27914ae129b83c237d03d68fec002646d1163f69/images/BigNASimulatedXYPlane.png" alt="Simulated intensity field XY plane Big NA" /></td>
      <td style="text-align: center"><img src="https://github.com/xiang-ji-ncsu/xiang-ji-ncsu.github.io/raw/27914ae129b83c237d03d68fec002646d1163f69/images/BigNADetectedXYPlane.png" alt="Experimental detected intensity field" /></td>
    </tr>
  </tbody>
</table>

<p>We did have direct proof if we modified the design to have the light field mainly in XY plane, but this one didn’t have the property to generate a sharper focus. (Small Numerical Aperture)</p>

<table>
  <thead>
    <tr>
      <th style="text-align: center">XY intensity plane simulated</th>
      <th style="text-align: center">intensity experimentally detected</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td style="text-align: center"><img src="https://github.com/xiang-ji-ncsu/xiang-ji-ncsu.github.io/raw/27914ae129b83c237d03d68fec002646d1163f69/images/SmallNASimulated.png" alt="Simulated intensity field XY plane Small NA" /></td>
      <td style="text-align: center"><img src="https://github.com/xiang-ji-ncsu/xiang-ji-ncsu.github.io/raw/27914ae129b83c237d03d68fec002646d1163f69/images/SmallNADetected.png" alt="Detected intensity field XY plane Small NA" /></td>
    </tr>
  </tbody>
</table>

<p>=======================================================</p>

<p>*All of the files in this page are copyrighted . They are provided for your convenience, yet you may download them only if you are entitled to do so by your arrangements with various publishers.</p>

<p>**Copyright statement copied from Dr. Jiasen Zhang’s webpage.</p>

    </div>



  </div>



  <footer id="footer">
    <p class="copyright">Copyright &copy; 2018 Xiang Ji's Webpage. Powered by <a href="http://jekyllrb.com">Jekyll</a>, theme by <a href="https://github.com/mattvh/solar-theme-jekyll">Matt Harzewski</a></p>
  </footer>



  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.1/jquery.min.js"></script>
  <script src="/assets/js/jquery.mobilemenu.min.js"></script>

  <script>
    $(document).ready(function(){
      $('#sidebar nav ul').mobileMenu({'topOptionText': 'Menu', 'prependTo': '#sidebar nav'});
    });
  </script>



	<!-- Google Analytics -->
	<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
		ga('create', 'UA-69626748-1', 'auto');
		ga('send', 'pageview', {
		  'page': '/Publication/',
		  'title': 'Research Projects'
		});
	</script>
	<!-- End Google Analytics -->

</body>
</html>